### DISCUSSION TITLE
Error when compiling chapter1

### DISCUSSION THREAD

**Discussion by ricardo02amaral** [joaomlourenco/novathesis#518] [Q&A] [answered]
Hello,
I have already installed Miktex and TexStudio and I have version 7.2.1 of ‘novathesis’ and when I compile and run template.tex it works fine but when I try to run chapter1.tex I get this error and I don't know how to solve it. 
By the way, the directory path is correct, so that can't be the problem.
![Captura de ecrã 2024-11-21 000018](https://github.com/user-attachments/assets/e2b67c2d-1952-41fd-997a-e5cf7b21b25f)

Thank you!

**Comment by joaomlourenco** [comment, marked as answer]
You cannot compile just chapter 1. You **always** have to compile the full document.

But you can select which chapters are included ubiquitous the document in file `Config/4_files.tex`.
